### DISCUSSION TITLE
Manual Installation?

### DISCUSSION THREAD

**Discussion by TXTFox** [ampache/ampache#3132] [General]
Is there anyway to manually install this? We are having major issues trying to get the db imported/created. It keeps giving us credential issues even tho all is fine - isnt inserting the db.

As seen here:
Unable to connect to the database: SQLSTATE[HY000] [1045] Access denied for user '#####_#######'@'localhost' (using password: YES)

**Comment by kuzi-moto** [comment]
Typically using the installer is the best way. You could go ahead and manually import the tables using the `resources/sql/ampache.sql` I believe. Then just make sure to have already created a database user.

Just to make sure though, you're using the root database admin account to create it, right? I've only run into issues if I use a different account.

**Reply by lachlan-00** [reply]
Manual install without the web interface is pretty possible but not something i've done myself.

the sql file is a valid sql file so you can import into a database
```
mysql -u root -p ampache < ./resources/sql/ampache.sql
```

copy the config file
```
cp config/ampache.cfg.php.dist config/ampache.cfg.php
```

update the config file with database host, name user and pass.
![image](https://user-images.githubusercontent.com/1305249/148167777-ee403887-6d51-4b99-abd2-dbd1e09c9d23.png)

then you should be able to create a user using the CLI
```
bin/cli admin:addUser admin -e email@gmail.com -n "some guy" -l 100
```
